tests/test_wave_05.py

```diff
@@ -71,28 +69,44 @@ def test_create_goal(client):
         }
     }
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
+    # pass
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={
+        "title": "A very important goal"
+    })
+    response_body = response.get_json()
 
     # Assert
     # ---- Complete Assertions Here ----
     # assertion 1 goes here
+    assert response.status_code == 200
     # assertion 2 goes here
+    assert "goal" in response_body
     # assertion 3 goes here
+    assert response_body == {
+        "goal": {
+            "title": "A very important goal",
+            "id": 1
+        }
+    }
     # ---- Complete Assertions Here ----
 
```

Comment:
We also want to query the database and assert that the goal has been updated in the database.